Objects/Converters/ConverterAutocadCivil/ConverterAutocadCivilShared/ConverterAutocadCivil.Civil.cs

```diff
@@ -1128,13 +1122,155 @@ public Pipe PipeToSpeckle(PressurePipe pipe)
 
   // corridors
   // this is composed of assemblies, alignments, and profiles, use point codes to generate featurelines (which will have the 3d curve)
+
+  private Base AppliedSubassemblyParamToSpeckle(IAppliedSubassemblyParam param)
+  {
+    Base baseParam = new() { 
+      ["name"] = param.DisplayName,
+      ["key"] = param.KeyName, 
+      ["type"] = param.ValueType.Name, 
+      ["value"] = param.ValueAsObject,
+      ["designValue"] = param.DesignValueAsObject};
+
+    return baseParam;
+  }
+
+  private CivilAppliedSubassembly AppliedSubassemblyToSpeckle(AppliedSubassembly appliedSubassembly)
+  {
+    // retrieve subassembly name
+    Subassembly subassembly = Trans.GetObject(appliedSubassembly.SubassemblyId, OpenMode.ForRead) as Subassembly;
+
```

Comment:
This one let's save for another issue to add `subassembly` objects, as this property should be on a new `Subassembly` class instead of the `AppliedSubassembly` class